### DISCUSSION TITLE
Innr RSL 110 / 115 from Hue Bridge to Zigbee2MQTT

### DISCUSSION THREAD

**Discussion by hawaltie** [Koenkk/zigbee2mqtt#9295] [Devices]
I want to move 2 Innr 110 / 115 sets (consisting of 3 bulbs each) from a Hue Bridge to Zigbee2MQTT. For some reason the factory reset (6 times off/on) doesn't work. Am I missing a point here?

**Comment by hawaltie** [comment]
Somebody a clue?

**Comment by francisp2** [comment]
Make sure the hue bridge is not powered on, or they will reconnect with it instead of Zigbee2mqtt

**Comment by hawaltie** [comment]
Tried that already, but it still doesn work. Perhaps I have to wait several hours until the hue network (I have many Hue bulbs) stops.
